README.md

```diff
@@ -292,16 +290,22 @@ HttpSysBinding has some limitations:
 * The Launcher has access problems when writing to the keyring on Linux if run as a service.
   The launcher configuration parameter `useLegacyDataProtection` needs to be set to `true` as a workaround. ([#100](https://github.com/fiskaltrust/middleware-launcher/issues/100)
 
+<!--
```

Comment:
We don't want these headings to be hidden from the README generally. They should be only removed in the pipeline when we pack the README in the zip.

I suggest you add comment like `<!-- trim-end --!>` here and in the pipeline remove everything after that comment.

Reply:
sure, I am going to add comments in README and a script that will throw out the appropriate version of md file